Cargo.toml

```
@@ -11,7 +11,7 @@ categories = ["wasm", "gui", "web-programming"]
 
 [dependencies]
 log = "0.4"
-dioxus = { version = "0.4.0", default-features = false, features = ["macro", "html"] }
+dioxus = { version = "0.5", default-features = false, features = ["launch", "macro", "html"] }
```

Do we need to add the launch feature in the library?
```suggestion
dioxus = { version = "0.5", default-features = false, features = ["macro", "html"] }
```

---

See https://github.com/DioxusLabs/dioxus/issues/2162

---

Damn, alright. Opened https://github.com/DioxusLabs/dioxus/pull/2171 and it also made me think and opened https://github.com/DioxusLabs/dioxus-std/issues/27